src/Button/button.stories.mdx

```diff
@@ -112,3 +122,7 @@ Buttons with components don't currently support icons
     <Button component={<a href="javascript:void()">With component</a>} />
   </Story>
 </Preview>
+
+# API
+
+<Props of={Button} />
```

Comment:
Nice! It would be great to setup this for all components in the future.

Reply:
Will do. Let's get some time during the cooldown to work on improving docs. I would like to also look for storybook alternatives.